src/cmds/hardware/i2c_tools/i2cdump/i2cdump.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ *
+ * @date Nov 16, 2018
+ * @author Anton Bondarev
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <stddef.h>
+
+#include <drivers/i2c/i2c.h>
+
+static void print_usage(void) {
+	printf("Usage: i2cdump -h\n");
+	printf("Usage: i2cdump [-r first-last] [-y] i2cbus address [mode]\n");
+}
+
+static void print_error(void) {
```

Comment:
Is it really necessary to use a function for it?

Reply:
I think it's may has more information in future